cmd/porter/copy.go

```diff
@@ -19,8 +19,8 @@ Destination can be either a registry, a registry/repository, or a fully tagged b
 If the source bundle is a digest reference, destination must be a tagged reference.
 `,
 		Example: `  porter bundle copy
-  porter bundle copy --source deislabs/porter-bundle:v0.1.0 --destination portersh
-  porter bundle copy --source deislabs/porter-bundle:v0.1.0 --destination portersh --insecure-registry
+  porter bundle copy --source getporter/porter-bundle:v0.1.0 --destination portersh
```

Comment:
For our documentation strings in the cli help text, we should make sure to use bundles that actually exist so that if they copy/pasted them, they worked. I don't believe that `porter-bundle` is a published bundle though which is a problem. How about we switch to `getporter/hello` as the bundle which is already published?